components/authentication/org.wso2.carbon.analytics.idp.client/src/test/java/org/wso2/carbon/analytics/idp/client/LocalIdPClientTest.java

```diff
@@ -196,4 +254,36 @@ public void testGetUserFailure() throws IdPClientException {
                 Collections.singletonList(defaultRole));
         Assert.assertNull(localIdPClient.getUser("admin2"));
     }
+
+   @Test(expectedExceptions = AuthenticationException.class)
```

Comment:
Looks like its not properly formatted run autoformatting

Reply:
The issue was addressed in the following commit https://github.com/wso2/carbon-analytics-common/pull/414/commits/e2da05d81422324b4a3ae8ccce4352975b9852b1